tests/testproject/TestProject.IntegrationServiceA/Oracle/OracleDatabaseExtensions.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Oracle.ManagedDataAccess.Client;
+
+public static class OracleDatabaseExtensions
+{
+    public static void MapOracleDatabaseApi(this WebApplication app)
+    {
+        app.MapGet("/oracledatabase/verify", VerifyOracleDatabaseAsync);
+    }
+
+    private static async Task<IResult> VerifyOracleDatabaseAsync(OracleConnection connection)
```

Comment:
This should be

```suggestion
    private static async Task<IResult> VerifyOracleDatabaseAsync(MyDbContext dbContext)
```

And then using the EF Component you are adding in this PR.